android/assets/jsons/translations/Dutch.properties

```diff
@@ -588,8 +569,7 @@ A [greatPerson] has been born in [cityName]! = [cityName] - Een [greatPerson] is
 We have encountered [civName]! = We zijn [civName] tegengekomen!
 [cityStateName] has given us [stats] as a token of goodwill for meeting us = [cityStateName] heeft ons [stats] gegeven als teken van welwillendheid om ons te ontmoeten
 [cityStateName] has given us [stats] as we are the first major civ to meet them = [cityStateName] heeft ons [stats] gegeven omdat we de eerste grote beschaving zijn die hen ontmoeten
- # Requires translation!
-[cityStateName] has also given us [stats] = 
+[cityStateName] has also given us [stats] = [cityStateName] heeft ons ook gegeven [stats]
```

Comment:
Ik zou het zelf in een andere volgorde zetten, `[cityStateName] heeft ons ook [stats] gegeven`